scalableminds/webknossos
# Persist skeleton group expansion state

https://moveskeletonexpansiontostore.webknossos.xyz/
### Steps to test:
- Go to skeletons tab, make sure to have some trees, ordered in some groups
- Expand/collapse groups and subgroups, make sure everything works well
- reload site and check that the same groups are expanded/collapsed, except root group

### TODOs:
  - [x] Add `isExpanded` to nml
      - [x] read
      - [x] write
  - [x] Improve NML Support
- [x] fix bug where empty groups cant be dragged

### Issues:
- fixes #

------
(Please delete unneeded items, merge only when none are left open)
